sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_helpers.py

```diff
@@ -0,0 +1,52 @@
+# coding=utf-8
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+
+from datetime import time
+from azure.core.exceptions import HttpResponseError
+
+POLLING_INTERVAL = 1
```

Comment:
We should honor any `Retry-After` headers present in the response on the polling URL. And one second interval is likely too aggressive. I believe @maririos is following up with the service team.

Reply:
Honoring the `Retry-After` header is baked into the `LROBasePolling` from azure.core. If it's there, we use it. Will sync with @samvaity to see if she knows a more appropriate polling interval, though.

Reply:
Per Cognitive Services guidelines, `Retry-After` should always return the interval. Most services do.
Are you seeing this is not the case and that is why we need an extra polling interval?

Reply:
Chris has told us to use 5s in the absence of a Retry-After header value so I'll update to that.